Solutions/Darktrace/Analytic Rules/CreateAlertFromSystemStatus.yaml

```diff
@@ -14,20 +14,35 @@ triggerThreshold: 0
 tactics: # none
 relevantTechniques: # none
 query: |
-  darktrace_model_alerts_CL
+  darktrace_model_alerts_CL //anything starting with 'Dt' is not an ASIM mapping 
   | where dtProduct_s =="System Alert"
-  | extend EventVendor = "Darktrace"
-  | extend EventProduct = "Enterprise Immune System"
+  | extend EventVendor="Darktrace", EventProduct="Darktrace DETECT"
+  | project-rename ThreatCategory=dtProduct_s, EventStartTime=time_s, NetworkRuleName=friendlyName_s, SrcIpAddr=deviceIP_s, SrcHostname=hostname_s, ThreatRiskLevel=priority_code_d, ThreatRiskCategory=priority_s, DtSeverity=Severity, DtName=name_s, DtStatus=status_s, DtMessage=Message, DtURL=url_s, DtUUID=uuid_g
 eventGroupingSettings:
   aggregationKind: AlertPerResult
-entityMappings: # none
+entityMappings:
+  - entityType: Host
+    fieldMappings:
+      - identifier: HostName
+        columnName: SrcHostname
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: SrcIpAddr
 customDetails:
-  Title: friendlyName_s
-  EventMessage: Message
+  EventStartTime: EventStartTime
+  NetworkRuleName: NetworkRuleName
+  ThreatRiskLevel: ThreatRiskLevel
+  ThreatRiskCategory: ThreatRiskCategory
+  DtSeverity: DtSeverity
+  DtName: DtName
+  DtStatus: DtStatus
+  DtMessage: DtMessage
+  DtURL: DtURL
 alertDetailsOverride:
-  alertDisplayNameFormat: 'Darktrace: {{friendlyName_s}}'
-  alertDescriptionFormat: "{{Message}}"
+  alertDisplayNameFormat: 'Darktrace: {{{NetworkRuleName}}'
```

Comment:
@JustasZDT, please check if the extra "{" is needed before NetworkRuleName, thanks.

Reply:
This is not needed and will also be removed, thanks.